docker-compose.yml

```diff
@@ -160,3 +160,21 @@ services:
     depends_on:
       - handshake
       - handshake-api
+
+  synckeys:
+    build:
+      context: docker/synckeys
+      dockerfile: Dockerfile
+    container_name: synckeys
+    restart: on-failure
+    logging: *default-logging
+    volumes:
+      - ./docker/data/keys:/keys
+      - ./docker/data/sia:/sia-data
+    environment:
+      - SIAD_IP=10.10.10.10
```

Comment:
this would also not be necessary using the `healthcheck` suggested below

Reply:
It turns out I don't even need the `healthcheck` for this! Changed.